src/compute/tests/table_v2_materialize.rs

```diff
@@ -117,17 +117,17 @@ async fn test_table_v2_materialize() -> Result<()> {
     ];
 
     // Create table v2 using `CreateTableExecutor`
-    let mut create_table = CreateTableExecutor::new(
-        source_table_id,
-        source_manager.clone(),
-        table_columns,
-        "CreateTableExecutor".to_string(),
-        Info::TableSource(Default::default()),
-    );
-    // Execute
-    create_table.open().await?;
-    create_table.next().await?;
-    create_table.close().await?;
+    // let mut create_table = CreateTableExecutor::new(
```

Comment:
Just delete them if not used.

Reply:
FYI, I fixed this test in this PR: https://github.com/singularity-data/risingwave/pull/2550